avocado/plugins/nrun.py

```diff
@@ -61,6 +61,14 @@ def configure(self, parser):
                                  parser=parser,
                                  long_arg='--status-server')
 
+        settings.register_option(section="nrun.podman",
+                                 key="enabled",
+                                 default=False,
+                                 key_type=bool,
+                                 help_msg="Spawn tests in podman containers",
+                                 parser=parser,
+                                 long_arg="--podman")
```

Comment:
If the previous comment it is valid, then maybe also rename here to `--spawn-on-podman` or `--podman-spawner`.